.travis.yml

```diff
@@ -51,6 +51,12 @@ matrix:
           SETUPOPTS=""
           PYLINTOPTS="--rcfile=ci/3.6.pylintrc"
           PYLINTFILES="azurelinuxagent setup.py makepkg.py tests"
+    - python: 3.9
```

Comment:
I just checked, we haven't been running coverage at all since Kevin added the helper scripts. I'm in favour of removing it completely as I never found much use for it... thoughts?

Reply:
haha wow! Now that you mention it, I do agree haven't seen that either in a while lol. 
I dont mind either tbh, I personally dont check it a lot but maybe the external contributors do? Not sure

Reply:
i am inclined to remove it

Reply:
I won't make any changes to `.travis.yml` outside Py 3.9 since we are replacing it with GitHub Actions.